detect_variants/variants_to_table.cwl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "SelectVariants (GATK 3.6)"
+baseCommand: ["/usr/bin/java", "-Xmx4g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "VariantsToTable"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 6000
+      tmpdirMin: 25000
+arguments:
+    ["-o", { valueFrom: $(runtime.outdir)/variants.tsv }]
+inputs:
+    reference:
+        type: File
+        inputBinding:
+            prefix: "-R"
+            position: 1
+        secondaryFiles: [".fai", "^.dict"]
+    vcf:
+        type: File
+        inputBinding:
+            prefix: "--variant"
+            position: 2
+        secondaryFiles: [.tbi]
+    fields:
+      type:
+        type: array
+        items: string
```

Comment:
I agree but I wasn't sure if that would work since I have these nested `type:` and `inputBinding:` to keep the array of args grouped on the command line.